services/motion/builtin/builtin_test.go

```diff
@@ -287,13 +288,13 @@ func TestPositionalReplanning(t *testing.T) {
 		// 	expectedSuccess: false,
 		// 	extra:           map[string]interface{}{"replan_cost_factor": 0.01, "smooth_iter": 5},
 		// },
-		{
-			name:            "check we replan with a noisy sensor",
-			noise:           r3.Vector{Y: epsilonMM + 0.1},
-			expectedErr:     fmt.Sprintf("exceeded maximum number of replans: %d: plan failed", 4),
-			expectedSuccess: false,
-			extra:           map[string]interface{}{"replan_cost_factor": 10.0, "max_replans": 4, "smooth_iter": 5},
-		},
+		// {
```

Comment:
Test shouldn't be disabled

Reply:
whoops, sorry about that. I had it commented out when I was investigating a failure in this test earlier.